browser-test/src/support/ti_dashboard.ts

```diff
@@ -0,0 +1,50 @@
+import {Page} from 'playwright'
+
+// This class is to test Civiform's TI to add new clients
+// It requires the tests to be logged as a TI
+export class TIDashboard {
+  public page!: Page
+
+  constructor(page: Page) {
+    this.page = page
+  }
+
+  async gotoTIDashboardPage(page: Page) {
+    await page.click('text="Trusted intermediary dashboard"')
+  }
+
+  async createClient(client: ClientInformation) {
+    await this.page.fill('label:has-text("Email Address")', client.emailAddress)
+    await this.page.fill('label:has-text("First Name")', client.firstName)
+    await this.page.fill('label:has-text("Middle Name")', client.middleName)
+    await this.page.fill('label:has-text("Last Name")', client.lastName)
+    await this.page.fill('label:has-text("Date Of Birth")', client.dobDate)
+    await this.page.click('text="Add"')
+  }
+
+  async expectDashboardContainClient(client: ClientInformation) {
+    expect(
+      `.cf-admin-question-table-row:has-text("${this.convertToMMDDYYYY(
+        client.dobDate,
+      )}")`,
+    )
+    expect(`.cf-admin-question-table-row:has-text("${client.emailAddress}")`)
+    expect(`.cf-admin-question-table-row:has-text("${client.firstName}")`)
+    expect(`.cf-admin-question-table-row:has-text("${client.lastName}")`)
+  }
+
+  convertToMMDDYYYY(dobDate: string): string {
+    const [year, month, day] = dobDate.split('-')
+    return `${month}-${day}-${year}`
+  }
+}
+
+// This class helps to test the TrustesIntermediary dashboard changes
+// We should be logged as a TI to use this class
```

Comment:
issue (blocking): this comment is identical to the one on the class

Reply:
Updated